templates/repository/common/CONTRIBUTING.md

```diff
@@ -13,37 +13,38 @@ https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Introduction](#introduction)
-- [FAQ](#faq)
-- [How can I contribute?](#how-can-i-contribute)
-- [Communication](#communication)
-- [Contribute examples](#contribute-examples)
-- [Contribute code](#contribute-code)
-- [Contribute documentation](#contribute-documentation)
-- [Disclosing vulnerabilities](#disclosing-vulnerabilities)
-- [Code style](#code-style)
-  - [Working with forks](#working-with-forks)
-- [Conduct](#conduct)
+- [Contribute to Ory $PROJECT](#contribute-to-ory-project)
+  - [Introduction](#introduction)
+  - [FAQ](#faq)
+  - [How can I contribute?](#how-can-i-contribute)
+  - [Communication](#communication)
+  - [Contribute examples](#contribute-examples)
+  - [Contribute code](#contribute-code)
+  - [Contribute documentation](#contribute-documentation)
+  - [Disclosing vulnerabilities](#disclosing-vulnerabilities)
+  - [Code style](#code-style)
+    - [Working with forks](#working-with-forks)
+  - [Conduct](#conduct)
```

Comment:
This shouldn't get updated in a pull request that just fixes typos. Can you please revert this part?

Reply:
Not sure how that happens, but I think it's prettier in VSCode? 

It automatically does this when I save the file... So this might be correct, actually? Do you know where this is coming from?